src/scss/darkmode_variables.scss

```diff
@@ -0,0 +1,89 @@
+
+/* EXPERIMENTAL */
+.dark-mode {
+    /* Background Color */
+    --oc-grey-900: #f8fafc;
```

Comment:
All changes looks nice, however doesn't grey-900 look more lightrer that grey-50, maybe grey-25 would be better name of color?

Reply:
dark mode is still experimental, either we use dark: utility from tailwind or just change the css variable.